skills/cc-web-setup/scripts/test-web-bootstrap.sh

```
@@ -0,0 +1,272 @@
+    && fail "codex fresh-login: token LEAKED into argv" \
+    || ok "codex fresh-login: token never in argv"
+}
+
+test_codex_blank_token() {
+  local d log envf; d="$(new_stub_dir)"; log="$WORK/c4.log"; envf="$WORK/c4-envfile"; : > "$envf"
+  # codex stub drops a sentinel if it is ever invoked.
+  write_stub "$d" codex "> '$WORK/c4-called'; exit 0"
+  rm -f "$WORK/c4-called"
+  # A whitespace-only token is non-empty (passes the -z guard) but unusable: it
+  # must be diagnosed as blank, NOT piped to `codex login --with-access-token`.
+  ( export CLAUDE_ENV_FILE="$envf"; run_ensure_codex_auth "$d" "$log" "   " ) \
```

Shellcheck (0.11.0)

[info] Modification of CLAUDE_ENV_FILE is local (to subshell caused by (..) group).

(SC2030)